java/libphonenumber/src/com/google/i18n/phonenumbers/nano/Phonemetadata.java

```diff
@@ -2,6 +2,8 @@
 
 package com.google.i18n.phonenumbers.nano;
 
+// @@protoc_insertion_point(imports_scope)
```

Comment:
This class is not hand-generated.

Did you generate the changes? Or were these added manually?